### DISCUSSION TITLE
Dependency graph

### DISCUSSION THREAD

**Discussion by Zaijo** [stackrox/kube-linter#512] [Q&A]
Hello Team, 
I've noticed that kube-linter allows for dependency check e.g. when service doesn't find any matching pod by labels.

Is it possible to build & export the dependency map somehow?
I'd like to use the dependency map as part of our static-site generated documentation.

**Comment by janisz** [comment]
I think you are referring to `Dangling Services` check. I'm afraid kube-linter currently does not graph output for this check.
